src/components/Sum/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export const Sum = () => ();
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```

Comment:
this wrapper is redundant
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
```